docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -35,3 +38,24 @@ Tiered Storage uses the following:
 - Data Persistence and Tiered Storage are mutually exclusive features. If both are enabled simultaneously, `InvalidConfigurationException` is thrown during the member startup.
 
 - SQL is not supported for the Tiered-Storage-backed maps. `UnsupportedOperationException` is thrown when SQL query is executed. To avoid this, use xref:query:predicate-overview.adoc[Predicate API] instead.
+
+== Supported Devices
+
+Hazelcast supports only NVMe (local) SSD devices for Tiered Storage.
+While HDDs and some network storages work with the same file abstraction that local SSDs do, and technically speaking such setups work, the access latency of these devices compared to that of the modern SSDs are so high that using Tiered Storage with such devices are not supported.
+
+== High-Level Design Overview
+
+The Tiered Storage feature for IMaps is backed by a special-purpose, addressable data structure called HybridLog.
```

Comment:
```suggestion
The Tiered Storage feature for IMaps is backed by a special-purpose, addressable data structure called HybridLog.
It is called HybridLog because it seamlessly combines in-place updates with a traditional append-only log.
```

Reply:
Added in https://github.com/hazelcast/hz-docs/pull/1047/commits/0253a16cd6c5ede529c5f0f74b0e730fe62b5d95